cime_config/testdefs/ExpectedTestFails.xml

```diff
@@ -30,6 +30,13 @@
 
   <!-- aux_clm test suite failures -->
 
+  <test name="ERS_D.f10_f10_mg37.I1850Clm51Sp.cheyenne_intel.clm-Meier2022_surf_rough">
+    <phase name="RUN">
+      <status>FAIL</status>
+      <issue>Error: Forcing height is below canopy height for patch index</issue>
+    </phase>
+  </test>
+
```

Comment:
Removing these lines because I have not encountered a RUN failure with this test.

Reply:
Awesome, that the test is working. So resolving.